src/uct/tcp/sockcm/sockcm_iface.c

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017-2019.  ALL RIGHTS RESERVED.
+ * Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include "sockcm_iface.h"
+#include "sockcm_ep.h"
+#include <uct/base/uct_worker.h>
+#include <ucs/sys/string.h>
+#include <ucs/sys/sock.h>
+
+enum uct_sockcm_process_event_flags {
+    UCT_SOCKCM_PROCESS_EVENT_DESTROY_SOCK_ID_FLAG = UCS_BIT(0),
+    UCT_SOCKCM_PROCESS_EVENT_ACK_EVENT_FLAG       = UCS_BIT(1)
+};
+
+static ucs_config_field_t uct_sockcm_iface_config_table[] = {
+    {"BACKLOG", "1024",
+     "Maximum number of pending connections for a (listening?) socket.",
+     ucs_offsetof(uct_sockcm_iface_config_t, backlog), UCS_CONFIG_TYPE_UINT},
+
+    {"SOCK_ID_QUOTA", "64",
+     "How many sockcm connections can progress simultaneously.",
+     ucs_offsetof(uct_sockcm_iface_config_t, sock_id_quota), UCS_CONFIG_TYPE_UINT},
+
+    {NULL}
+};
+
+static UCS_CLASS_DECLARE_DELETE_FUNC(uct_sockcm_iface_t, uct_iface_t);
+
+static ucs_status_t uct_sockcm_iface_query(uct_iface_h tl_iface,
+                                           uct_iface_attr_t *iface_attr)
+{
+    memset(iface_attr, 0, sizeof(uct_iface_attr_t));
+
+    iface_attr->iface_addr_len  = sizeof(ucs_sock_addr_t);
+    iface_attr->device_addr_len = 0;
```

Comment:
agreed

Reply:
@dmitrygladkov leaving this and the next feedback back you have as is. Only because I'm following what is already the case on other UCTs. If there's a strict nececssity to change this, I'll do that.

Reply:
It's OK then :)